src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -311,6 +314,14 @@ public Task CopyToAsync(Stream stream)
             return CopyToAsync(stream, null);
         }
 
+#if NET46
+        // Workaround for HttpWebRequest synchronous resubmit
```

Comment:
Ah, saw it now. Since you know the cause, if you have some time, please update the comment.